xendit-android/src/main/java/com/xendit/Xendit.java

```diff
@@ -162,6 +162,11 @@ private void createSingleOrMultipleUseToken(final Card card, final String amount
                 return;
             }
 
+            if (card.getCreditCardCVN() != null && !CardValidator.isCvnValidForCardType(card.getCreditCardCVN(), card.getCreditCardNumber())) {
```

Comment:
`card.getCreditCardCVN() != null` i think is redundant. doesnt `isCvnValidForCardType` hanldle it already?

Reply:
this is the same like iOS, we don't want to validate if there is no CVN input